packages/examples/package.json

```diff
@@ -10,25 +11,23 @@
   "files": [
     "examples/"
   ],
+  "workspaces": [
+    "examples/*"
```

Comment:
Since this package is now private, how is the `examples` package intended to be worked with? Do you have to `cd` into this directory manually and run `yarn setup`, etc.? Sorry if this is already explained somewhere.

Reply:
Better question: does marking this as `private: true` exclude it from the set of workspaces Yarn iterates over for commands such as `yarn workspaces foreach`, etc.? Or is it the fact that the examples here are sub-workspaces?

Reply:
The `examples` package is still part of the workspace, but we want to make is a sub-workspace (where the individual example folders are part of the workspace), so I marked it as `private`. It doesn't change the behaviour of `workspaces foreach` for the `examples` package, but making it a sub-workspace will make it run the commands in the individual example folders too now.

Reply:
Gotcha 👍🏻